fhirflat/resources/extensions.py

```diff
@@ -417,6 +417,114 @@ def elements_sequence(cls):
         ]
 
 
+class Race(_DataType):
+    """
+    An ISARIC extension collecting data on the race of a patient.
+    """
+
+    resource_type: str = Field(default="Race", const=True)
+
+    url: str = Field("race", const=True, alias="url")
+
+    valueCodeableConcept: fhirtypes.CodeableConceptType = Field(
+        None,
+        alias="valueCodeableConcept",
+        title="Value of extension",
+        description=(
+            "Value of extension - must be one of a constrained set of the data "
+            "types (see [Extensibility](extensibility.html) for a list)."
+        ),
+        # if property is element of this resource.
+        element_property=True,
+        element_required=True,
+    )
+
+    @classmethod
+    def elements_sequence(cls):
+        """returning all elements names from
+        ``Extension`` according specification,
+        with preserving original sequence order.
+        """
+        return [
+            "id",
+            "extension",
+            "url",
+            "valueCodeableConcept",
+        ]
+
+
+class presenceAbsence(_DataType):
+    """
+    An ISARIC extension to indicate if a clinical finding is present, absent or unknown.
+    """
+
+    resource_type: str = Field(default="presenceAbsence", const=True)
+
+    url: str = Field("presenceAbsence", const=True, alias="url")
+
+    valueCodeableConcept: fhirtypes.CodeableConceptType = Field(
+        None,
+        alias="valueCodeableConcept",
+        title="Value of extension",
+        description=(
+            "Value of extension - must be one of a constrained set of the data "
+            "types (see [Extensibility](extensibility.html) for a list)."
+        ),
+        # if property is element of this resource.
+        element_property=True,
+        element_required=True,
+    )
+
+    @classmethod
+    def elements_sequence(cls):
+        """returning all elements names from
+        ``Extension`` according specification,
+        with preserving original sequence order.
+        """
+        return [
+            "id",
+            "extension",
+            "url",
+            "valueCodeableConcept",
+        ]
+
+
+class prespecifiedQuery(_DataType):
+    """
+    An ISARIC extension to indicate if a finding is the result of a prespecified query.
```

Comment:
What would be an example of this? No need to update the PR - but will be good to have a list of extensions with examples.